src/connector/src/utils.rs

```diff
@@ -51,6 +52,10 @@ impl Properties {
     pub fn get_kinesis(&self, key: &str) -> Result<String> {
         self.get_inner(key, " when using Kinesis source")
     }
+
+    pub fn get_connector_type(&self) -> Result<String> {
+        self.get_inner(UPSTREAM_SOURCE_KEY, "when get connector")
+    }
```

Comment:
```suggestion
    pub fn get_connector_type(&self) -> Result<String> {
        const UPSTREAM_SOURCE_KEY: &str = "connector";
        self.get_inner(UPSTREAM_SOURCE_KEY, " for connector type")
    }
```

We can totally hide UPSTREAM_SOURCE_KEY this key.

Reply:
fixed